### DISCUSSION TITLE
Fast Fourier Transform/Discrete Fourier Transform

### DISCUSSION THREAD

**Discussion by Jamiewhbee** [Part-II-Computational-Physics/.github#5] [Q&A]
What's the spacing dk in reciprocal space for the discrete Fourier transform in terms of the spacing dx in real space? 
I would expect something like dk = 2 * pi/dx but that suggests dk diverges in the limit of the continuous Fourier series (n -> infinity).
Since I expect dk should be of order 1/n (the same as dx), maybe it's more like dk = 2 * pi/(n^2 * dx)?

**Comment by AustenLamacraft** [comment]
The spacing is $dk=2\pi/L$ with $L=Ndx$. You can have a finite $dk$ even when $dx\to 0$, if $N\to\infty$ so that $L$ is fixed.
